grapple/types/images.py

```diff
@@ -59,6 +60,15 @@ def get_rendition_type():
     return rendition_type
 
 
+def rendition_allowed(rendition_filter):
+    """Checks a given rendition filter is allowed"""
+    allowed_filters = getattr(settings, "GRAPPLE_ALLOWED_IMAGE_FILTERS", None)
+    if allowed_filters is None or not isinstance(allowed_filters, list):
```

Comment:
```suggestion
    if allowed_filters is None or not isinstance(allowed_filters, (list, tuple)):
```

suggestion: it doesn’t cost much so we should also protect people from shooting themselves in the foot by using a tuple instead of the expected list (which would allow all filters with the current implementation).